injector/network_disruption.go

```diff
@@ -70,6 +71,12 @@ type tcServiceFilter struct {
 	priority uint32 // one priority per tc filters applied, the priority is the same for all interfaces
 }
 
+// tcFilter describes a tc filter
+type tcFilter struct {
```

Comment:
Originally I wanted to re-use tcServiceFilter, but the contents of `networkDisruptionService` were a little too coupled with what the k8s api returns when describing a filter, so I thought it was best to define a smaller type with exactly what I need